arrow-fx-test/build.gradle

```diff
@@ -8,15 +9,13 @@ plugins {
 }
 
 apply from: "$SUBPROJECT_CONF"
+apply from: "$PUBLISH_CONF"
 
 dependencies {
     compile project(":arrow-fx")
-    compile "io.arrow-kt:arrow-core-test:$VERSION_NAME", excludeArrow
+    api "io.arrow-kt:arrow-core-test:$VERSION_NAME", excludeArrow
     compile "io.arrow-kt:arrow-mtl:$VERSION_NAME", excludeArrow
-    compile "io.arrow-kt:arrow-mtl-data:$VERSION_NAME", excludeArrow
     compile "io.arrow-kt:arrow-free:$VERSION_NAME", excludeArrow
-    compile "io.arrow-kt:arrow-free-data:$VERSION_NAME", excludeArrow
-    compile "io.arrow-kt:arrow-recursion-data:$VERSION_NAME", excludeArrow
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
     compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
```

Comment:
Created #93 for the removal of this dependency. This is low priority so not blocking for 0.10.x